File: trismart_smartboard_integration/test_client/rpc_test.py
# from xmlrpc import client
import odoorpc

db_name = 'trismart-staging-15-0-3640780'
username = 'admin'
api_key = 'TriSMART2021'
server_url = ['trismart-staging.odoo.com', 80]

sb_lead_id = 5204497
x_api_key = '281a27b3f1865a80f6d0acd5fb307cbf'
project_template_id = 45

odoo = odoorpc.ODOO(server_url[0], port=server_url[1])
# odoo = odoorpc.ODOO(server_url)
odoo.login(db_name, username, api_key)

user_data = odoo.execute('smartboard.connector', 'create_project', sb_lead_id, x_api_key, project_template_id)
# user_data = odoo.execute('smartboard.connector', 'create_project', sb_lead_id, x_api_key)
print(user_data)

File: trismart_smartboard_integration/test_client/rpc_test.php
<?php

require_once ('vendor/autoload.php');
use OdooRPCClient\Client;


$data = [
   'host' => 'https://trismart-staging-15-0-3640780.dev.odoo.com',
   'database' => 'trismart-staging-15-0-3640780',
   'login' => 'admin',
   'password' => '0c5608e57939826227c64338c981070573de0700'
];


$odoo = new Client($data['host']);
$odoo->login($data['database'],$data['login'],$data['password']);

$sb_lead_id = 6;
$project_template_id = 45;
$x_api_key = 'bWFnZ2llOnN1bW1lcnM';

$new_project = $odoo->env['smartboard.connector']->create_project($sb_lead_id , $x_api_key, $project_template_id);
var_dump($new_project);

?>
